client/index.html

```diff
@@ -1,11 +1,473 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <template data-type="pug" data-src="./view/base/meta.pug" />
-    <link rel="stylesheet" href="./index.css">
-    <script type="module" src="./index.js" defer></script>
-  </head>
-  <body>
-    <template data-type="pug" data-src="./view/browsers/browsers.pug" />
-  </body>
+<head>
+  <link rel="stylesheet" href="./index.css">
+  <script type="module" src="./index.js" defer></script>
+  <meta name="color-scheme" content="dark light">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+</head>
+
+<body>
+  <article class="Article Article--intro">
+      <div class="Article--intro__header">
+        <h1 class="Article__h1">
+          <img src="./assets/browserlist_logo.svg" role="presentation">
+          Browserslist
+        </h1>
+        <a class="Badge" href="https://github.com/browserslist/browserslist">
+          <img src="./assets/github_logo.svg" role="presentation">
+          10.5k
+        </a>
+        <a class="Badge" href="https://twitter.com/Browserslist">
+          <img src="./assets/twitter_logo.svg" role="presentation"> 4k
+        </a>
+      </div>
+      <p class="Article--intro__about">Shared browser and platform compatibility config for popular JavaScript tools,
+        including
+        Autoprefixer, Babel, ESLint, PostCSS, and Webpack</p>
+      <p class="Article--intro__supported">
+        Supported by
+        <img src="./assets/evil_maritians_logo.svg" role="presentation">
+        <a href="" class="Link">Evil Martians</a> and
```

Comment:
Bad practice to keep links without real URLs. It is ready to miss, so I always try to add real address when I write any link.

Reply:
We still have `href=""` in source code

Reply:
No...
https://github.com/browserslist/browserl.ist/pull/306/files#diff-edfa4829c58ad696e48ab606527eabead1471963a743486a94b65fe215035814R30
<img width="730" alt="image" src="https://user-images.githubusercontent.com/27442574/183062941-dd8c453d-ab50-4899-913f-c7b200226727.png">

Reply:
Hm. Strange GitHub UX bug.